modules/oe_content_sub_entity/modules/oe_content_sub_entity_extra_authors/src/EventSubscriber/SubEntityExtraAuthorsSubscriber.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\oe_content_sub_entity_extra_authors\EventSubscriber;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\oe_content_sub_entity\SubEntitySubscriberBase;
+
+/**
+ * Event subscriber for handing entity labels for "Author" entity type bundles.
+ */
+class SubEntityExtraAuthorsSubscriber extends SubEntitySubscriberBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function generateLabel(ContentEntityInterface $entity): ?string {
+    $label = NULL;
+    switch ($entity->bundle()) {
+      case 'oe_person':
+      case 'oe_organisation':
+        $label = $this->defaultLabel($entity);
```

Comment:
We don't need 
```
      case 'oe_person':
      case 'oe_organisation':
        $label = $this->defaultLabel($entity);
```

anymore since  `case 'oe_person':, case 'oe_organisation':` are default cases